build.gradle

```diff
@@ -336,8 +369,40 @@ dependencies {
 
   smokeTestImplementation sourceSets.main.runtimeClasspath
 
-  integrationTestImplementation sourceSets.main.runtimeClasspath
```

Comment:
Why are these 2 lines of Code removed.Integration test should stay put as they are...This task should only put in values,variables and code for contractTests.